crates/client/db/src/rocksdb_options.rs

```diff
@@ -13,22 +60,30 @@ pub fn rocksdb_global_options() -> Result<Options> {
     let mut options = Options::default();
     options.create_if_missing(true);
     options.create_missing_column_families(true);
-    let cores = std::thread::available_parallelism().map(|e| e.get() as i32).unwrap_or(1);
-    options.increase_parallelism(cores);
-    options.set_max_background_jobs(cores);
 
+    // See module documentation.
     options.set_atomic_flush(true);
-    options.set_max_subcompactions(cores as _);
 
+    // By default rocksdb will spam a lot of info about every column very regularily, making huge files that take
+    // multiple gigabytes. Using log level warn, this info will not be printed to the log file. In addition,
+    // we limit the size and number of files.
```

Comment:
I think you intended this below for the `set_log_level`?

Reply:
i mean that for the 3 following settings, but i'll swap the lines so that the log level warn is just below this comment and the other two (which is what this "In addition" refers to) follow just after that